packages/template-retail-react-app/app/constants.js

```diff
@@ -34,6 +34,26 @@ export const HOME_SHOP_PRODUCTS_LIMIT = 10
 export const CAT_MENU_DEFAULT_NAV_SSR_DEPTH = 1
 export const CAT_MENU_DEFAULT_ROOT_CATEGORY = 'root'
 
+//Constants for default product badges
+export const PRODUCT_TILE_BADGE_LABELS = [
+    {
+        propertyName: 'c_isNew',
+        label: {
```

Comment:
We want to use define Message here to make sure all the messages are extracted into our json even though it may or may not use at once at run time. 
```suggestion
        label: defineMessege({
           id: '...',
           defaultMessage: 'New'
        })
```

Reply:
The CLI script (`npm run build-translations`) only does the static analysis of the code. It's not smart enough to understand `formatMessage(item.label)` and know what `item.label` dynamically refers to.

So yeah, like Alex wrote, we'll need call `defineMessage` first.